sql-odbc/build_win_debug64.ps1

```diff
@@ -1,2 +1,6 @@
 $WORKING_DIR = (Get-Location).Path
+$env:VCPKG_DEFAULT_TRIPLET = 'x64-windows'
+cd src
+vcpkg install
```

Comment:
is there any sanity check would could log in case vcpkg install fails?

Reply:
Not too sure how we would accomplish this nicely. Build errors will output to `CMakeOutput.log` and `CMakeOutput.log`. `Vcpkg` errors will output to STDOUT and CMake will fail to find the libraries. Users can check their `vcpkg` logs but that can be dependent on how and where `vcpkg` is installed. We could output STDOUT to some file for `vcpkg`, but that doesn't seem overly useful imo.